docs/reference/data-frames/apis/delete-transform.asciidoc

```diff
@@ -22,7 +24,11 @@ Deletes an existing {dataframe-transform}.
 `data_frame_transform_id` (required)::
   (string) Identifier for the {dataframe-transform}.
 
-//===== Authorization
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have
+`manage_data_frame_transforms` cluster privileges to use this API.
```

Comment:
The cluster privilege is `manage_data_frame_transforms` a role that has that privilege is `data_frame_transforms_admin`. 

I am not sure if we document the specific privileges or the roles we created ourselves.

Reply:
We didn't put the built-in roles in the ML APIs, but I think it's helpful so I've added it here